streams/src/main/java/org/apache/kafka/streams/state/Stores.java

```diff
@@ -44,6 +44,40 @@
 
 /**
  * Factory for creating state stores in Kafka Streams.
+ * <p>
+ * When using the high-level DSL, i.e., {@link org.apache.kafka.streams.StreamsBuilder StreamsBuilder}, users create
+ * {@link StoreSupplier}s that can be further customized via
+ * {@link org.apache.kafka.streams.kstream.Materialized Materialized}.
+ * For example, a topic read as {@link org.apache.kafka.streams.kstream.KTable KTable} can be materialized into an
```

Comment:
is the second `KTable` intentional?

Reply:
That is correct:

```
{@link org.apache.kafka.streams.kstream.KTable KTable}
```

will show on java docs as `org.apache.kafka.streams.kstream.KTable`, while the above will show as `KTable` whose ref links to `org.apache.kafka.streams.kstream.KTable` still.